contrib/k8s/examples/postgresql/advanced/postgresql-database-instance.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: servicecatalog.k8s.io/v1beta1
+kind: ServiceInstance
+metadata:
+  name: my-postgresql-database-instance
```

Comment:
Within the same example, we should stick with the same naming convention, either my- or example-.